config/locales/en.yml

```diff
@@ -118,7 +118,7 @@ en:
     flash:
       applications:
         create:
-          notice: 'Application created.'
+          notice: 'Application created with Secret: %{application_secret}'
```

Comment:
I don't remember, but the page contains only the table with applications without secrets, right? 
I'm just thinking is that a good way to show new app secret..

Also let's user lower `s` :)

Reply:
Exactly, right now it would look like this:

![Screenshot 2019-04-29 at 09 14 00](https://user-images.githubusercontent.com/3407787/56881010-4004bb00-6a5f-11e9-99dc-b33ac60beece.png)

The other way I could think of would be to redirect with the application secret as URL Parameter so that the `show` action could still show the secret in this place at least one time after creating the application.

But still, after refresh it would be gone and The user may not be aware of this. At least, if the secret is in the flash, its more intuitive that its gone after the flash is gone.

I could also imagine to make the flash text more descriptive, e.g. add that this is the only time the secret is shown (but maybe only if `hash_application_secrets` is enabled?)?



> Also let's user lower s :)

Done 👍

Reply:
Well, in this case I prefer to store secret using flashes like:

```ruby
flash[:application_secret] = ...
```
and then something like the following on the UI:

```ruby
<%= flash[:application_secret].presence || @application.secret %>
```

So not to show it in the notice, but in *Secret* section where it must be :)

---

> But still, after refresh it would be gone and The user may not be aware of this.

It's absolutely OK, many services do it in such way (GitHub for it's two-factor auth as well)

Reply:
Ohh, alright, I didn't know you could do it this way, thanks for letting me know!

I'll go back and do it the way you suggested!